mythril_core/src/acpi.rs

```diff
@@ -0,0 +1,84 @@
+#![deny(missing_docs)]
+use core::fmt;
+
+/// Extented BIOS Data Area Start Address
+const EXTENDED_BIOS_DATA_START: usize = 0x0000040e;
+/// Extented BIOS Data Area End Address
+const EXTENDED_BIOS_DATA_END: usize = 0x0000040e + 2048;
+/// Main BIOS Data Area Start Address
+const MAIN_BIOS_DATA_START: usize = 0x000e0000;
+/// Main BIOS Data Area End Address
+const MAIN_BIOS_DATA_END: usize = 0x000fffff;
+/// Well Known RSDP Signature
+const RSDP_SIGNATURE: &[u8; 8] = b"RSD PTR ";
+
+/// The Root System Descriptor Pointer (RSDP)
+#[derive(Copy, Clone, Debug)]
```

Comment:
I think this is a bit too big to derive Copy

Reply:
:+1: